src/services/app-config.service.ts

```diff
@@ -1,14 +1,16 @@
-import axios from "axios";
-import getEnvApi from "./get-env-api";
-import store from "@/store";
-import { fetchAllApps, fetchApp } from "./dashboard.service";
-import bytes from "bytes";
+/* eslint-disable */
```

Comment:
Disabled eslint for this file for now. Will delete this file once redesign is completed